lichess-bot.py

```diff
@@ -926,10 +955,15 @@ def start_lichess_bot() -> None:
     parser.add_argument("-v", action="store_true", help="Make output more verbose. Include all communication with lichess.")
     parser.add_argument("--config", help="Specify a configuration file (defaults to ./config.yml).")
     parser.add_argument("-l", "--logfile", help="Record all console output to a log file.", default=None)
+    parser.add_argument("--disable_auto_logging", action="store_true", help="Disable automatic logging.")
     args = parser.parse_args()
 
     logging_level = logging.DEBUG if args.v else logging.INFO
-    logging_configurer(logging_level, args.logfile)
+    auto_log_filename = None
+    if not args.disable_auto_logging:
+        now = datetime.datetime.now()
+        auto_log_filename = f"./log/{now.year}-{now.month}-{now.day}_{now.hour}-{now.minute}-{now.second}.txt"
```

Comment:
The file names will sort incorrectly without zero-padding the parts of the date.

`auto_log_filename = now.strftime("./log/%Y-%m-%d_%H-%M-%S.txt")`

This is probably the best place to change the log folder name (see above comments for line 139).